.buildkite/pipeline.yml

```diff
@@ -59,7 +75,10 @@ steps:
     async: true
 
   - label: 'Trigger React Native CLI pipeline'
-    depends_on: "publish-js"
+    depends_on:
+# TODO: Reinstate later
+#      - 'publish-js'
```

Comment:
Is later now or even later?

Reply:
Even later - I realised as I was checking through that I should have made it cleaner.  It should happen naturally though when we come in install and run the CLI.